src/main/proto/protocol0.proto

```diff
@@ -36,33 +36,11 @@ message Metric {
   repeated Attribute attributes = 7;
 }
 
-message Event {
-  // time in ms when the event was generated.
-  optional int64 time = 1;
-  // key of event.
-  optional string key = 2;
-  // value of event.
-  optional double value = 3;
-  // host where event originated.
-  optional string host = 4;
-  // indicated state of this event.
-  optional string state = 5;
-  // description of event.
-  optional string description = 6;
-  // time this event should be considered valid in seconds.
-  optional int64 ttl = 7;
-  // tags associated to event.
-  repeated string tags = 8;
-  // attributes associated to event.
-  repeated Attribute attributes = 9;
-}
-
 message Attribute {
   required string key = 1;
   optional string value = 2;
 }
 
 message Message {
   optional Metric metric = 1;
-  optional Event event = 2;
```

Comment:
ah actually maybe i shouldn't remove this to remain backwards compatible? not sure how ffwd uses this.

Reply:
Yeah, its pretty much never a good idea to remove a protobuf field. You can annotate it with `[deprecated=true]`.

Reply:
https://github.com/spotify/ffwd/blob/master/modules/protobuf/src/main/java/com/spotify/ffwd/protobuf/ProtobufDecoder.java#L111

I could do https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Or making it a reserved field - https://developers.google.com/protocol-buffers/docs/proto#reserved